packages/styleguide/src/components/App/App.js

```diff
@@ -164,37 +192,32 @@ const GlobalStyle = createGlobalStyle`
   *::after { box-sizing: border-box; }
   *::before { box-sizing: border-box; }
 `;
-/* eslint-enable */
 
-const PageLayout = styled.div``;
+const Overlay = styled('div')`
+  .is-active & {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    z-index: 10;
+    background-color: ${props => props.theme.colors.overlay};
+  }
+`;
 
-const PageHeader = styled(Header)`
-  position: fixed;
-  top: 0;
-  right: 0;
-  left: 0;
-  max-width: 100%;
-  height: ${props => rem(props.theme.sizes.headerHeight)};
-  display: flex;
-  flex: 0 0 auto;
-  justify-content: space-between;
-  align-items: center;
-  padding: 0 ${props => rem(props.theme.spaces.large)};
-  background-color: ${props => props.theme.colors.main};
-  color: ${props => props.theme.colors.black};
-  z-index: ${props => props.theme.zIndex.header};
+const SitemapWrapper = styled('div')`
+  margin-top: ${props => props.theme.sizes.headerHeight};
 `;
 
+const PageLayout = styled.div``;
```

Comment:
Toto tu asi nemusi byt, je to tu kvoli citatelnosti?

Reply:
trochu som reorganizoval file, aby to bolo podla pouzitia a takto to bolo zadefinovane predtym, nechal som to pre citatelnost, nech je hned jasne, vymazat to?

Reply:
Ja osobne by som to zmazal

Reply:
ok